features/step_definitions/manage_user_account_spec.rb

```diff
@@ -0,0 +1,28 @@
+Given(/^a registered MAS user$/) do
+  step 'I register'
+end
+
+Given(/^that I am on the user account edit settings page in "(.*?)"$/) do |language|
+  settings_page.load(locale: 'en')
+
+  expect(page).to have_content(I18n.t('authentication.settings.title'))
+end
+
+When(/^I fill in the contact number$/) do
+  settings_page.contact_number.set '03005005000'
+  settings_page.current_password.set 'password'
+
+  settings_page.submit.click
+  #click_on I18n.t('authentication.settings.label')
```

Comment:
Sorry I feel like I'm giving you a hard time here ... but we should really remove this line.

Once that's done though it's a :+1: from me :)